requests/__init__.py

```diff
@@ -9,29 +9,22 @@
 Requests HTTP Library
 ~~~~~~~~~~~~~~~~~~~~~
 
-Requests is an HTTP library, written in Python, for human beings. Basic GET
-usage:
+Requests is an HTTP library, written in Python, for human beings.
+Basic GET usage:
 
    >>> import requests
    >>> r = requests.get('https://www.python.org')
    >>> r.status_code
    200
-   >>> 'Python is a programming language' in r.content
+   >>> b'Python is a programming language' in r.content
    True
 
 ... or POST:
 
    >>> payload = dict(key1='value1', key2='value2')
    >>> r = requests.post('https://httpbin.org/post', data=payload)
-   >>> print(r.text)
-   {
-     ...
-     "form": {
-       "key2": "value2",
-       "key1": "value1"
-     },
-     ...
-   }
+   >>> r.json()
```

Comment:
Hi, I would suggest to do so: 
```
   >>> print(r.text)
   ... # doctest: +ELLIPSIS
   ... # doctest: +NORMALIZE_WHITESPACE
   {
     ...
     "form": {
       "key2": "value2",
       "key1": "value1"
     },
     ...
   }
```
So we change the test minimally.

Reply:
Thanks. I did not know that I could use this syntax.

Reply:
Thanks. I did not know that I could use this syntax. I'm going to change it according to your suggestion.

Reply:
I added NORMALIZE_WHITESPACE and ELLIPSIS to the pytest.ini file and removed from the doctest in order to use them in pytest configuration.